### ISSUE TITLE
[Feature Request]Working Voice Dimensions with GT-MP

### ISSUE THREAD

**Issue by DasChaosAT** [AlternateLife/JustAnotherVoiceChat#7]
The Posibility to activate a dimension depending Voice System, which works with the GT-MP Dimension.
For Example:
Player from Dimension 1 can hear Player from Dimension 0, but not otherwise. But in Dimension -1 nobody can hear you and you can't hear nobody from an other Dimension. It should work like the Dimension Player Streaming habbit from GT-MP.

**Comment by markatk** [comment]
Good call, we totally forgot dimensions. I am not sure yet, if the GT-MP dimension system is the best approach till we want to be independent of a single game, but we will introduce some kind of dimension system. Probably the GT-MP server wrapper will wrap the dimensions to match the GT-MP behavior if we decide to go with a different system.
